great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1968,7 +2552,12 @@ def _create_df_for_charting(
 
         if plot_mode is PlotMode.PRESCRIPTIVE:
             for kwarg_name in expectation_configuration.kwargs:
-                df[kwarg_name] = expectation_configuration.kwargs[kwarg_name]
+                if isinstance(expectation_configuration.kwargs[kwarg_name], list):
```

Comment:
Is it always a list? Or should we check for other iterables like tuples?

Reply:
This is an oddity of `pandas` wanting to split the list into columns (when we want it to be a single column for now). I'm not sure that the behavior would be the same for other iterables and I don't see documentation on it.